### PR TITLE
Add&Integrate pundit with organization

### PR DESCRIPTION
**Pull request by NikolayGGorbunov**

### CODE REVIEW

File: app/policies/package_policy.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      if user.admin?
+        scope.where('user_id' => user.organization.users.ids).order('created_at desc')
+      else
+        scope.where('user_id' => user.id)
+      end
+    end
+  end
+
+  def index?
+    (@user.admin? || @user.operator?) && @user.organization.present?
```
Comment (Sh1mensoka) [inline app/policies/package_policy.rb:20]: Why do wee need to explicitly check if a user an admin or operator?

File: spec/controllers/packages_controller_spec.rb [hunk 2]
```diff
@@ -5,17 +5,29 @@
...
       expect(subject).to redirect_to(new_user_session_path)
     end
   end
 
-  describe '#index' do
-    subject(:sort) { get :index, params: sort_params }
+  shared_examples 'when user dont have org' do
+    it 'raise error' do
+      sign_in user2
+      expect(subject).to redirect_to(root_path)
```
Comment (Sh1mensoka) [inline spec/controllers/packages_controller_spec.rb:16]: Example says that it raises error, but I can't see any error in expectations

Comment (NikolayGGorbunov) [reply spec/controllers/packages_controller_spec.rb]: Fixed

File: app/policies/package_policy.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+class PackagePolicy < ApplicationPolicy
+  class Scope < Scope
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
+      if user.admin?
+        scope.where('user_id' => user.organization.users.ids).order('created_at desc')
```
Comment (Sh1mensoka) [inline app/policies/package_policy.rb:12]: nit: "rocket is dead"
next time use more modern and conventional way to describe query to DB 
example: where(user_id: user.organization.users.ids)

**Review by Sh1mensoka — APPROVED** [review]

**Review by NadezhdaGlin — APPROVED** [review]

**Pull request merged**